assets/section-footer.css

```diff
@@ -337,24 +342,22 @@
 
```

Comment:
The email in the footer needs to have an underline, this will target only hyperlinks within the rich text editor of the footer blocks, the safest way to target this base element is that won't have a class on it.

```suggestion
.footer-block__details-content .rte > a {
  text-decoration: underline;
}
```
The following is what it should look like:

<img width="437" alt="Screenshot 2024-06-06 at 9 41 25 AM" src="https://github.com/Applied-Labs-LLC/al-quickfire-test/assets/5162080/86dc3857-b680-4ded-b9d7-40dd49c2bfd3">